### DISCUSSION TITLE
Fooocus 2.3 lightning preset and Upscale2X amazing results

### DISCUSSION THREAD

**Discussion by wisdombox44** [lllyasviel/Fooocus#2578] [Show and tell]
New fooocus 2.3 lightning preset and Upscaler works amazing. People talking and testing about new SUPIR upscale. My personal experience Fooocus Upscaler out performs and does better than most of the upscalers. The image below generated with this settings
Model: juggernautXL_v9
Preset: Initial
Performance: lightning
![2024-03-20_19-36-42_7495](https://github.com/lllyasviel/Fooocus/assets/123548315/0fdcf59d-ce04-4014-96e8-5f1f43ebb146)

When I upscale with Upscale (2x)
with same settings but random seed,
here is the result with amazing details
![2024-03-20_19-42-26_5995](https://github.com/lllyasviel/Fooocus/assets/123548315/95e5e2af-16fb-4121-bc6f-8e7a778f360e)

The interestig thing is
I switch to Performance : Quality when Upscale (2x) details not much good and generation time is longer
here is the quality upscale result
![2024-03-20_19-40-17_7733](https://github.com/lllyasviel/Fooocus/assets/123548315/ce9d6256-a265-4b3d-a53a-b63e3bd29410)

Long story short New Lightning Preset an Upscale works perfect without changing any settings and preset.
Thank you Guys Fooocus is getting bettert and better in every update
